opentelemetry-etw-logs/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "opentelemetry-etw-logs"
+description = "OpenTelemetry-Rust exporter to ETW"
+version = "0.1.0"
+edition = "2021"
+homepage = "https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/opentelemetry-etw-logs"
+repository = "https://github.com/open-telemetry/opentelemetry-rust-contrib/tree/main/opentelemetry-etw-logs"
+readme = "README.md"
+rust-version = "1.65.0"
+keywords = ["opentelemetry", "log", "trace", "etw"]
+license = "Apache-2.0"
+
+[dependencies]
+tracelogging = "1.2.1"
+tracelogging_dynamic = "1.2.1"
+opentelemetry = { workspace = true, features = ["logs"] }
+opentelemetry_sdk = { workspace = true, features = ["logs"] }
+async-std = { version = "1.6" }
```

Comment:
what do we use `async-std` for?

Reply:
Legacy. Used `cargo-machete`: https://github.com/bnjbvr/cargo-machete.
```
nalyzing dependencies of crates in this directory...
cargo-machete found the following unused dependencies in C:\open-src\opentelemetry-rust-contrib\opentelemetry-etw-logs:
opentelemetry-etw-logs -- C:\open-src\opentelemetry-rust-contrib\opentelemetry-etw-logs\Cargo.toml:
        async-std
        chrono
```

Removing async-std and chrono